stats-test.cpp

```diff
@@ -14,7 +14,10 @@ TEST_CASE("reports average, minimum and maximum") {
 }
 
 TEST_CASE("average is NaN for empty array") {
-    auto computedStats = Statistics::ComputeStatistics({});
+	auto computedStats = Statistics::ComputeStatistics({});
```

Comment:
Resolve the mismatch between the description of the test case and the REQUIREs inside it